src/cimSupport/cmdletization/xml/cmdlets-over-objects.xsd

```diff
@@ -0,0 +1,1050 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I changed the default build of SMA to Debug. Makes for a much better debugging experience.
This change would be really nice to have merged. It makes working on the project much nicer.